### ISSUE TITLE
config.js.erb: undefined method `each' for nil:NilClass

### ISSUE THREAD

**Issue by gingaz** [thejandroman/puppet-kibana3#18]
Hi,

I'm experiencing difficulties installing kibana3. I simply add "include kibana3" and get the following error:
```
Error: Could not retrieve catalog from remote server: Error 400 on SERVER: Failed to parse template kibana3/config.js.erb:
  Filepath: /etc/puppet/environments/dev/modules/kibana3/templates/config.js.erb
  Line: 18
  Detail: undefined method `each' for nil:NilClass
```
I'm running puppet 3.7 with parser=future.
Any ideas what is wrong?

Cheers,
Gin

**Comment by thejandroman** [comment]
Hello, @gingaz. Unfortunately I haven't had a chance to test this module with the future parser on. The template looks fine, and init is inheriting from params corretly. I wonder if it's an issue with scoping in the template. I'll have to investigate a little to track it down.

**Comment by gingaz** [comment]
Hi thejandroman,

thanks for the response.

Indeed, the module works just fine with the default parser. 
This particular problem goes away if you refer to variables via  <%= scope.lookupvar('::var_refference') %>. Unfortunately other issues occurred afterwards. 

Cheers,
Gin

**Comment by thejandroman** [comment]
This was solved by PR#22
